core/src/main/resources/pl/messages.properties

```
@@ -1416,3 +1416,18 @@ coronastats.deaths.today=Zgonów dzisiaj:
 coronastats.critical=Krytycznych przypadków:
 coronastats.updated=Stan na
 coronastats.unknown.country=Nie ma takiego kraju! Przestań zmyślać kraje, użyj angielskiej nazwy lub skrótu ISO.
+
+ustawpoziom.help.description=Ustawia poziom uprawnień dla danej komendy
+ustawpoziom.help.uzycie=<list|set|remove> [komenda] [numeryczny poziom uprawnień]
+ustawpoziom.help.extended=\nPoziomy uprawnień znajdziesz pod komendą `permlevel`\nPrzykładowe użycie: \n```\nustawpoziom list\nustawpoziom set userinfo 2\nustawpoziom remove userinfo\n```
+ustawpoziom.list.desc=%s - %s (%s poziom)
+ustawpoziom.remove.doesnt.exist=Taka komenda nie ma zmienionego poziomu uprawnień!
+ustawpoziom.remove.succes=Przywrócono podstawowy poziom uprawnień dla tej komendy!
+ustawpoziom.set.badlvl=Taki poziom uprawnień nie istnieje!
+ustawpoziom.set.nopermissions=Prawdopodobnie próbujesz ustawić poziom dla komendy do, której nie masz permisji lub chcesz ustawić poziom większy od twojego
```

jak już bawimy się w interpunkcję-
```suggestion
ustawpoziom.set.nopermissions=Prawdopodobnie próbujesz ustawić poziom dla komendy, do której nie masz uprawnień, lub chcesz ustawić poziom większy od swojego
```